src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -175,10 +190,14 @@
   (react/create-element
     [:div {}
      [:div {:style {:position "absolute" :right 2 :top 2}}
-      [:div {:style {:backgroundColor (:button-blue style/colors) :color "#fff" :padding "0.5em" :cursor "pointer"}
+      [:div {:style {:backgroundColor
+                     (:button-blue style/colors)
```

Comment:
Don't split the key and value like this.  These style properties easily fit on 2 lines.